src/v/storage/types.h

```diff
@@ -263,6 +263,7 @@ struct truncate_config {
     truncate_config(model::offset o, ss::io_priority_class p)
       : base_offset(o)
       , prio(p) {}
+    // Lowest offset to remove.
```

Comment:
Shouldn't it be the lowest offset to preserve?

Reply:
No, IIUC it looks like:


[1.....10][11...13]

Reply:
🤦 thanks